### DISCUSSION TITLE
Paketo buildpack for newrelic how to specify you own newrelic.yml

### DISCUSSION THREAD

**Discussion by drgeb** [paketo-buildpacks/feedback#72] [General]
How do I specify and add my own newrelic.yml to the final docker image (so adding the newrelic.yml  to the newrelic layer) ?

**Comment by dmikusa** [comment]
Presently, you can't. It's not supported. There isn't a specific reason, just that the functionality hasn't been implemented. It could be added if necessary and someone contributes a PR. The primary challenge is that the file can contain secret information, which can't/shouldn't be included in the image (that would be an information leak).

For the time being, the expected workflow is to use bindings to set New Relic configuration. Bindings are secure key/values that you pass through to the application. A helper added by the buildpack translates them to `NEW_RELIC_<key>=<value>` environment variables just before your app starts and the New Relic agent reads them. Most (maybe all?) of the New Relic config can be set via an environment variable.
